### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by FrauDeborah**
Johnathan @haxwell 

While setting up a new computer for the backend, I notice a few corrections for the README.md file.

1. Changed instructions to tell people to use JDK8 rather than JDK11. Removed reference to JDK 9 as JDK9 is no longer an options on Oracle.

2. Added tools to set up including links for

   - GitHub (perhaps over kill ;0)  
   - Git download 
   - Postman download 
   - Eclipse IDE download

3. Corrected the instructions for forking, cloning and configuring the backend. (Note: No small wonder the new people were having issues with setting up.)

Please review and push to dev and master if you agree. Or let me know if more changes are needed.

Thank you,

Deborah @FrauDeborah

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .classpath
 .settings/
 /bin/
+popq2db.bat
```
Comment (haxwell) [inline .gitignore]: What application produces this file?

File: README.md [hunk 2]
```diff
@@ -25,10 +31,28 @@ These dependencies can be installed using the [Homebrew](https://brew.sh/) packa
...
+- Using Git Bash or Git CMD clone your newly forked repository to your local computer:
+    `$ git clone https://github.com/<<UseYourGitHubHandleHere>>/quizki-backend.git`
+
+- Change to the quizki-backend directory. 
+    `$ cd quizki-backend`
+    
+- Configure your local quizki-backend to use remote upstream. 
+    `$ git remote add upstream hhttps://github.com/team-quizki/quizki-backend`  
+    
+- Verify upstream set correctly
```
Comment (haxwell) [inline README.md]: Can we, in a few words, be more specific about what "set correctly" means? IE, what should the user expect to see?

**Comment by haxwell** [comment]
@FrauDeborah These are excellent additions to the README.. Thank you :) I have added a couple minor comments, but this PR is pretty much ready to merge.

**Comment by FrauDeborah** [comment]
Johnathan,

Do you mean this file? *popq2db.ba <http://popq2db.ba>*t

Well, I likely used notepad to write popq2db.bat. I use the file
popq2db.bat to repopulate the database, so I don't need to type the entire
line.

This is the content of the popq2db.bat  file.

rem re-populating quizki2_db for testing
mysql -u quizki2 -p <
./src/main/resources/META-INF/sql/populate_quizki2_db_20180619.sql

I guess since you asked, I shouldn't have created the bat.

Truth is I could not remember where I should store a bat file, so I just
put it in the project root thinking I would not be pushing to the backend.
Guess I was wrong....

Did you want me to remove it?

Regards,

Deborah Frauenfelder

**Comment by FrauDeborah** [comment]
Johnathan,

Good catch! I'll fix this tomorrow.

+    `$ git remote add upstream
hhttps://github.com/team-quizki/quizki-backend`
<http://github.com/team-quizki/quizki-backend>
+
+- Verify upstream set correctly

Can we, in a few words, be more specific about what "set correctly" means?
IE, what should the user expect to see?


Let me know where to put my popq2db.bat file, so I can correct both issues
at the same time if necessary.
Regards,

Deborah Frauenfelder

**Comment by haxwell** [comment]
@FrauDeborah RE: `popq2db.bat`, this is a script that you've created to automate a task. This is exactly what you should be doing, so no don't delete it! I took a look at the `.gitignore` and it seems the `/bin` directory is being ignored. Though I could see a future where we would -not- want to ignore `/bin`, it is for now, a safe place to put scripts that you create like `popq2db.bat`.

tl;dr: move `popq2db.bat` to `\bin`.

**Comment by FrauDeborah** [comment]
Thank you, Johnathan,

I’ll move popq2db.bat to the /bin directory, then I will remove popq2db.bat from the .gitignore file. Perhaps I should have asked in the first place. On second thought I should have considered the /bin myself — don’t know where my head was. /bin is for exactly that purpose and I saw /bin in the .gitignore —Dang.


Regards,

Deborah 


On Jan 6, 2019, at 12:16 AM, Johnathan James <notifications@github.com> wrote:

@FrauDeborah RE: popq2db.bat, this is a script that you've created to automate a task. This is exactly what you should be doing, so no don't delete it! I took a look at the .gitignore and it seems the /bin directory is being ignored. Though I could see a future where we would -not- want to ignore /bin for now, it is a safe place to put scripts that you create like popq2db.bat.

**Comment by FrauDeborah** [comment]
@Johnathan James <haxwell@gmail.com>
Hi Johnathan,

I didn't see a /bin in the quizki-backend directory structure. Where should
the /bin be placed in the directory structure?

Regards,

Deborah Frauenfelder

File: README.md (hunk 2 above)
Comment (FrauDeborah) [reply README.md]: Johnathan,

I added example output for the 'git remote -v' 

Hopefully that will work....

Deborah

Comment (FrauDeborah) [reply README.md]: Johnathan ( @haxwell ),

Previously, I updated  the text for showing a sample output for "git remote -v"

Today, I created the \bin at the root, moved the popq2db.bat to .\bin\popq2db.bat, and removed the popq2db.bat from the .gitignore. 

Let me know if you would like to see other changes for the quziki-backend README.md file. If no changes, then please merge the revised README.md file to dev (and perhaps master since it is also wrong.)

Deborah

**Comment by haxwell** [comment]
Great addition! Thanks!
